### ISSUE TITLE
changing .tc-config in git repo?

### ISSUE THREAD

**Issue by benlieb** [bernardop/iterm-tab-color-oh-my-zsh#6]
I'm a bit confused by the suggestion to change the .tc-config in the repo. Won't this constantly conflict with pulls?

Wouldn't it be more standard to put this in `~/.tc-config` ?

**Comment by walkerh** [comment]
I agree that using a location outside the repo is the standard. If you must have a file inside the repo directory that the user changes, you should normally add that file to `.gitignore` and than provide a sample version of the file for users to copy and edit.

Options:

- Use the existing location, but update the `.gitignore` file to exclude this file and `git mv .tc-config sample.tc-config`
- `${HOME}/.tc-config`  (less obvious to the user months later what this file is for)
- `${HOME}/.iterm-color-oh-my-zsh.config` (second best option for most software)
- `${XDG_CONFIG_HOME:-$HOME/.config}/iterm-color-oh-my-zsh/tc.config` (most standard compliant)
- `${XDG_DATA_HOME:-$HOME/.local/share}/iterm-color-oh-my-zsh/tc.config` (frequently used but not really for config)
- `${ZSH_CUSTOM}/config/iterm-color-oh-my-zsh/tc.config` (going your own way)

See:

- https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
- https://unix.stackexchange.com/questions/312988/understanding-home-configuration-file-locations-config-and-local-sha
- https://unix.stackexchange.com/questions/24347/why-do-some-applications-use-config-appname-for-their-config-data-while-other
- https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/zsh-navigation-tools#configuration
